src/RageFileManager.cpp

```diff
@@ -305,9 +305,9 @@ static RString ExtractDirectory( RString sPath )
 	return sPath;
 }
 
+#if defined(UNIX) || defined(MACOSX)
```

Comment:
Are there any Windows callers of this function? I think it's better to to have the API be consistent across platforms, o I think it's okay to just put the ifdef within the body.

Reply:
No, there are no Windows callers of this. This may have been a compiler warning about the function being unused on Windows, but I'll double check.

Reply:
Yeah, if I keep the `ifdef` within the body, MSVCC reports this warning, which gets treated as an error:

```
warning C4505: 'ReadlinkRecursive': unreferenced function with internal linkage has been removed
```